panther_manager/config/safety.xml

```diff
@@ -2,45 +2,72 @@
 <root BTCPP_format="4">
   <BehaviorTree ID="Safety">
     <Sequence>
-      <Sequence name="BatteryStatusSequence"
-                _skipIf="battery_health == POWER_SUPPLY_HEALTH_GOOD \
-|| battery_health == POWER_SUPPLY_HEALTH_UNKNOWN">
-        <SignalShutdown reason="Battery status unhealthy"/>
+      <Sequence name="BatteryOverheatSequence"
+                _skipIf="battery_health != POWER_SUPPLY_HEALTH_OVERHEAT">
+        <Sequence name="FatalBatTempSequence"
+                  _skipIf="bat_temp &lt;= FATAL_BAT_TEMP">
+          <SignalShutdown reason="Fatal battery temperature"/>
+        </Sequence>
+        <Sequence name="CriticalBatTempSequence"
+                  _skipIf="bat_temp &lt;= CRITICAL_BAT_TEMP">
+          <CallTriggerService name="TriggerEStop"
+                              service_name="hardware/e_stop_trigger"
+                              timeout="100"
+                              _skipIf="e_stop_state"/>
+          <CallSetBoolService name="DisableAUX"
+                              data="false"
+                              service_name="hardware/aux_power_enable"
+                              timeout="100"
+                              _skipIf="!aux_state"/>
+        </Sequence>
+        <Sequence name="HighBatTempSequence">
+          <CallSetBoolService name="EnableFanIfHighBatTemp"
+                              data="true"
+                              service_name="hardware/fan_enable"
+                              timeout="100"
+                              _skipIf="fan_state"/>
+        </Sequence>
```

Comment:
Actually,  HighBatTempSequence can be removed as all its behaviors can be encapsulated in BatteryOverheatSequence.